--- a/backend/Crypt/RSA.php
+++ b/backend/Crypt/RSA.php
@@ -458,13 +458,7 @@ class Crypt_RSA {
     function Crypt_RSA()
     {
         if ( !defined('CRYPT_RSA_MODE') ) {
-            switch (true) {
-                case extension_loaded('openssl') && version_compare(PHP_VERSION, '4.2.0', '>='):
-                    define('CRYPT_RSA_MODE', CRYPT_RSA_MODE_OPENSSL);
-                    break;
-                default:
-                    define('CRYPT_RSA_MODE', CRYPT_RSA_MODE_INTERNAL);
-            }
+			define('CRYPT_RSA_MODE', CRYPT_RSA_MODE_INTERNAL);
         }
 
         if (!defined('CRYPT_RSA_COMMENT')) {
