diff --git a/backend/webclient-sources/config/config-example.js b/backend/webclient-sources/config/config-example.js
index cfeda4e..0305f92 100644
--- a/backend/webclient-sources/config/config-example.js
+++ b/backend/webclient-sources/config/config-example.js
@@ -48,7 +48,7 @@ ClientConfig.storeVoteUrls      = serverinfos.tServerStoreVoteUrls; //do not use
 ClientConfig.getResultUrl      = server1url + 'getresult'; //?XDEBUG_SESSION_START=ECLIPSE_DBGP&KEY=13727034088813';
 
 // TODO make anonymize a config in backend/config
-ClientConfig.anonymizerUrl = 'http://anonymouse.org/cgi-bin/anon-www_de.cgi/'; // used to change the ip and to strip browser infos / with trailing slash
+ClientConfig.anonymizerUrl = ''; // used to change the ip and to strip browser infos / with trailing slash
 ClientConfig.voteClientUrl = server2url + 'getclient';
 
 ClientConfig.oAuth2Config = serverinfos.authModules.oAuth2;
